test/e2e/internal/utils/matcher.go

```diff
@@ -48,6 +54,15 @@ func (m *Matcher) MatchErrKeyWords(keywords ...string) *Matcher {
 	return m
 }
 
+// NotMatchKeyWords guarantees that the given keywords do not match with
```

Comment:
nit:
The func doc dose not align with the func name.

Reply:
Updated.